docs/scanner-quickstart.md

```diff
@@ -141,6 +141,11 @@ Scanner address: 0xAAA8C491232cB65a65FBf7F36b71220B3E695AAA
 Successfully initialized at /yourname/.forta
 ```
 
+**NEVER LOSE OR DELETE YOUR PRIVATE KEY in `~/.forta/.keys`.**
+
+!!! warning "Private Key"
+    Losing the private key will require actions to initiate a withdrawal for the old scan node address (takes 10 days), withdraw and deposit for the new scan node address.
```

Comment:
for security reasons, stake withdrawal takes 10 days and requires ownership of the staking shares.